pwm/pwm.sls

```diff
@@ -0,0 +1,110 @@
+#installs packages needed for PWM, downloads the WAR file and installs it into tomcat
+{%- from 'pwm/map.jinja' import data with context %}
+
+install_packages:
+  pkg.installed:
+    - names:
+      - java-1.8.0-openjdk
+      - tomcat
+      - wget
+      - unzip
+      - httpd
+      - inotify-tools
+      - s3cmd
+      - at
+      - postfix
+      - cyrus-sasl-plain
+      - chrony
+      - python-setuptools
+      - python2-pip
+
+#install_aws_cli:
+#  pip.installed:
+#    - name: awscli
+
+remove_unused_packages:
+  pkg.removed:
+    - names:
+      - ntp
+      - ntpdate
+
+retrieve_war_file:
+  cmd.run:
+    - name: aws s3 cp s3://{{ grains['config_bucket'] }}/files/pwm18.war /usr/share/tomcat/webapps/ROOT.war
```

Comment:
It seems not great to hardcode this S3 path to the file. Just take the whole path as the variable. Also, why is this not pillar?